processor/processor.go

```diff
@@ -2582,11 +2582,27 @@ func ConvertToFilteredTransformerResponse(events []transformer.TransformerEvent,
 	supportedMessageTypesCache := make(map[string]*cacheValue)
 	supportedMessageEventsCache := make(map[string]*cacheValue)
 
+	toAbortJobRunIDs := config.GetStringVar("", "RSources.toAbortJobRunIDs")
```

Comment:
Did not know the presence of such a method.(_How can I speak of that to which I have no knowledge_ 😛)
Tangentially, why do we even need a slice, just a `strings.Contains(config.GetString(abortDestID, jobParams.DestinationID)` would suffice, no? Considering `jobParams.DestinationID` can never be empty.
Also this would need some publishing as people are used to comma separated config probably